requirements/dev.txt

```diff
@@ -29,3 +29,7 @@ mypy~=1.1.0
 boto3-stubs[appconfig,serverlessrepo]>=1.19.5,==1.*
 types-PyYAML~=6.0
 types-jsonschema~=3.2
+
+# Check package version (backport importlib.metadata for python 3.7)
+# https://docs.python.org/3/library/importlib.metadata.html
+importlib_metadata; python_version < '3.8'
```

Comment:
EOL can't come quick enough

Reply:
It has EOL. GH will show a red sign if EOL is missing

Reply:
Actually, what if we instead fallback to previous behavior? So we don't add dependencies. I highly doubt they'll remove it before EOL.

Reply:
> It has EOL. GH will show a red sign if EOL is missing

https://endoflife.date/python

Reply:
I don't think it is a big deal and using metadata.version is more future proof. The backport is suggested from the official python doc (see code comment) and it is in dev.txt.

Reply:
> https://endoflife.date/python

LOL What I am thinking. EOF